src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.nio.file.Path;
+import java.util.Scanner;
+
+import duke.commands.Command;
+import duke.commands.CommandType;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Duke is a chatbot that helps you keep track of your tasks.
+ * In this version, Duke is renamed to LilBro.
+ */
+public class Duke {
+
+    /** The task list that is used to store the user's tasks. */
+    private final TaskList taskList = new TaskList();
+
+    /** The storage that is used to save and load the user's tasks. */
+    private final Storage storage;
+
+    /**
+     * Constructs a new Duke object.
+     *
+     * @param path The path to the save file.
+     */
+    public Duke(String path) {
+        Ui.greet();
+        Ui.println("Checking for a save file...");
+        String projectRoot = System.getProperty("user.dir");
```

Comment:
Can I ask why you used System.getProperty to get the current working directory?

Reply:
I don't recall exactly why I chose to do this, but it seems to be a common method to retrieve the current working directory.

https://stackoverflow.com/questions/4871051/how-to-get-the-current-working-directory-in-java
https://stackoverflow.com/questions/24709769/java-using-system-getpropertyuser-dir-to-get-the-home-directory